client/my-sites/preview/main.jsx

```diff
@@ -119,6 +119,11 @@ class PreviewMain extends React.Component {
 					showClose={ false }
 					previewUrl={ this.state.previewUrl }
 					externalUrl={ this.state.externalUrl }
+					loadingMessage={
+						this.props.translate( '{{strong}}One moment please…{{/strong}} loading your site.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '{{strong}}One moment please…{{/strong}} loading your new post.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).